src/components/ui/accordion.tsx

```diff
@@ -1,64 +1,75 @@
-import type * as React from "react";
```

Comment:
This change provides regression. IMo, I will create anotehr accordion just for new page, to avoid any changes. Astro does code splitting per page anyway.